safe_transaction_service/history/serializers.py

```diff
@@ -474,7 +474,10 @@ class SafeModuleTransactionResponseSerializer(GnosisBaseModelSerializer):
     transaction_hash = serializers.SerializerMethodField()
     block_number = serializers.SerializerMethodField()
     is_successful = serializers.SerializerMethodField()
-    module_transaction_id = serializers.SerializerMethodField()
+    module_transaction_id = serializers.SerializerMethodField(
+        help_text="Internal calculated parameter to unique identify a moduleTransaction \n"
+        "`ModuleTransactionId = i+tx_hash+trace_address`"
```

Comment:
I think for RST it should be `double quote`. Did you check this is rendered propertly on swagger? (same with \n)

Reply:
Also better `Internally calculated parameter to uniquely identify a moduleTransaction`

Reply:
yes I checked it, 
![image](https://github.com/safe-global/safe-transaction-service/assets/7888669/124a6523-2444-4647-a820-342228a81583)